data/templates/markdown/class.md.twig

```diff
@@ -66,13 +66,13 @@
 ### Details
 
 * File: [{{ node.path }}]({{ '../' ~ path(node.file) }})
-{% for interface in node.interfaces %}
+{% for interface in node.interfaces if interface.methods|length > 0 %}
 {% if loop.first %}
 * Implements:
 {% endif %}
   * [{{ interface }}]({{ link(interface) }})
 {% endfor %}
-{% for trait in node.usedTraits %}
+{% for trait in node.usedTraits if trait.methods|length > 0 %}
```

Comment:
I believe we need to use `methods(trait)` to attribute for inheritance. Also traits can have properties (constants too starting PHP `8.2`) and no methods, so maybe we need to check for that too?

Reply:
I checked that phpdocumentor v3 (our fork's template version) supports constants for traits - see here:
https://github.dev/phpDocumentor/phpDocumentor/blob/eec5cc7e0f384c603116dc6d759886bf1120048c/src/phpDocumentor/Descriptor/TraitDescriptor.php#L36

And here (trait):
https://github.dev/phpDocumentor/phpDocumentor/blob/eec5cc7e0f384c603116dc6d759886bf1120048c/src/phpDocumentor/Descriptor/Traits/HasConstants.php#L23-L62

So I added the check.

---

But for some reason (which I don't know), when I inspect the `TraitDescriptor.php` in the phar used in [DSAPI](https://github.com/Automattic/domain-services-client), it doesn't include these methods. I'm not familiar with phar files, but this might be something we want to check later. 

FWIW, I also [added a check for the method used internally within the Twig extension](https://github.com/Automattic/phpdoc-markdown/pull/2/files#diff-504b094b23ce85994c6ff1838c633ee4adf2aa13c7a501b71902ff5cc4e2caa3R75) anyway.